models/engine/file_storage.py

```diff
@@ -68,3 +68,25 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def count(self, cls=None):
+        '''count objects'''
+        count = 0
+        if cls is None:
+            universe = self.all()
+        else:
+            universe = self.all(cls)
+        for uni in universe:
+            count += 1
+        return count
```

Comment:
Please, comment get method arguments and give a brief or generic explanation about get method